README.md

```diff
@@ -1,6 +1,22 @@
 # aider-environment
 
-run
+## note about docker
+
+If you are not familliar with docker, is better before running script from this repository ensure, that you have configured and running docker. Try run command:
+```
+$ docker ps
+CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES
+```
+If you are got message as above, all is good and you can continue to section [run](#run).
+
+If you are got something like:
+```
+$ docker ps
+permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.45/containers/json": dial unix /var/run/docker.sock: connect: permission denied
+```
+you must add current user to group `docker` and relogin current session.
```

Comment:
@srbt how about:

you will need to add your current user to the `docker` group and log in again to your current session.